WhateverGreen/kern_rad.hpp

```diff
@@ -129,6 +129,23 @@ class RAD {
 	 *  Max framebuffer base functions per kext
 	 */
 	static constexpr size_t MaxGetFrameBufferProcs = 3;
+	
+	/**
+	 *  Variables and functions needed to enable Amd Navi10 pwm backlight control
```

Comment:
I would like to have better documentation. Ideally per each variable, and per each solved symbol pair.